picard.spec

```diff
@@ -85,8 +100,17 @@ coll = COLLECT(exe,
 
 
 if platform.system() == 'Darwin':
-    info_plist = {'NSHighResolutionCapable': 'True',
-                  'NSPrincipalClass': 'NSApplication'}
+    info_plist = {
+        'NSHighResolutionCapable': 'True',
+        'NSPrincipalClass': 'NSApplication',
+        'CFBundleName': 'picard',
+        'CFBundleDisplayName': 'MusicBrainz Picard',
+        'CFBundleGetInfoString': 'Audio file tagger',
+        'CFBundleIdentifier': 'org.musicbrainz.picard',
+        'CFBundleVersion': macos_picard_version,
+        'CFBundleShortVersionString': macos_picard_version,
+        'NSHumanReadableCopyright': 'Copyright MetaBrainz Foundation',
```

Comment:
First "Copyright" should not be part of this string, otherwise it will display "Copyright: Copyright MetaBrainz Foundation" in the info dialog (see my previous screenshot). 

And second I am not sure we should set this, as it is actually not really correct because there are multiple copyright holders (all Picard contributors). So either we duplicate the copyright statement from the about dialog or we just leave this away entirely. In the latter case looking at the info of the app with Finder the Copyright line will not even be shown.

Reply:
Removed, it was just to test.